### PR TITLE
Update iconfont for up arrow in chat inbox divider

### PR DESCRIPTION
**Pull request by akalin-keybase**
Use up arrow in chat divider.

Also clarify iconfont instructions.

### CODE REVIEW

**Comment by akalin-keybase** [comment]
r @keybase/react-hackers

File: shared/images/iconfont/README.md [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
 1. turn off subsetting
 1. truetype hinting = keep existing
+1. check the agree then download
+1. replace `kb.ttf` with the file of the same name from the downloaded zip file
 
-## Update the flowtype and fonts on both apps
+## Phase 5: Update the flowtype and fonts on both apps
 
-1. Go to /shared
+1. Go to `shared/`
```
Comment (akalin-keybase) [inline shared/images/iconfont/README.md]: This phase didn't seem to do anything. Did I do something wrong?

File: shared/chat/inbox/row/divider.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const Divider = ({isExpanded, isBadged, toggle}: DividerProps) => (
   >
     <DividerBox>
       <Box style={_dividerIconStyle}>
-        <Icon type={isExpanded ? 'iconfont-keybase' : 'iconfont-down-arrow'} inheritColor={true} />
+        <Icon type={isExpanded ? 'iconfont-up-arrow' : 'iconfont-down-arrow'} inheritColor={true} />
```
Comment (akalin-keybase) [inline shared/chat/inbox/row/divider.js]: When is the up arrow here actually used? It looks like when I click on the down arrow, this Divider turns into the FloatingDivider below.

**Comment by keybase-ci-visdiff** [comment]
The commits bed5ef31ea68...28076813def6 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Icon-Icon_Icon_Font** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504152295219-bed5ef31ea68-28076813def6-linux/Icon-Icon_Icon_Font-bed5ef31ea68.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504152295219-bed5ef31ea68-28076813def6-linux/Icon-Icon_Icon_Font-28076813def6.png)  
   Icon-Icon_Icon_Font comparison
</summary>

File: shared/images/iconfont/README.md (hunk 1 above)
Comment (cjb) [reply shared/images/iconfont/README.md]: I think it didn't do anything for me last time I did the update either.

Comment (akalin-keybase) [reply shared/images/iconfont/README.md]: Cool, updated doc to mention that

**Comment by keybase-ci-visdiff** [comment]
The commits bed5ef31ea68...95f34226347d introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **Icon-Icon_Icon_Font** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504155344301-bed5ef31ea68-95f34226347d-linux/Icon-Icon_Icon_Font-bed5ef31ea68.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1504155344301-bed5ef31ea68-95f34226347d-linux/Icon-Icon_Icon_Font-95f34226347d.png)  
   Icon-Icon_Icon_Font comparison
</summary>

File: shared/chat/inbox/row/divider.js (hunk 2 above)
Comment (chrisnojima) [reply shared/chat/inbox/row/divider.js]: If you have a lot of simple and big teams it'll show up. The floating divider shows if the big teams are below the fold, but if the big teams are above the fold then the regular divider shows (with the up arrow)

Comment (chrisnojima) [reply shared/chat/inbox/row/divider.js]: sim

**Comment by chrisnojima** [comment]
I'm going to merge this as i need it

**Pull request merged**